spec/abstractops.html

```diff
@@ -1154,6 +1154,7 @@ <h1>ParseISODateTime ( _isoString_ )</h1>
       1. Let _minute_ be the part of _isoString_ produced by the |TimeMinute|, |TimeMinuteNotValidDay|, |TimeMinuteThirtyOnly|, or |TimeMinuteThirtyOneOnly| productions, or *undefined* if none of those are present.
       1. Let _second_ be the part of _isoString_ produced by the |TimeSecond| or |TimeSecondNotValidMonth| productions, or *undefined* if neither of those are present.
       1. If the first code unit of _year_ is 0x2212 (MINUS SIGN), replace it with the code unit 0x002D (HYPHEN-MINUS).
+      1. If year is "-000000", throw a *RangeError* exception.
```

Comment:
This is missing a bit of formatting:
```
1. If _year_ is *"-000000"*, ...
```
(The formatting is used for metadata purposes in the rendered text, such as highlighting all uses of a variable.)